### ISSUE TITLE
How to review the loos of a particular datapoint(train image)

### ISSUE THREAD

**Issue by eravallirao** [keras-team/keras#14002]
Hi,
Is there a way to print the loss of a training datapoint along with it its filename? Can someone please help me with this.
Thanks

**Comment by Saduf2019** [comment]
@eravallirao 
Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
